WebAPI/Controllers/ParcelController.cs

```diff
@@ -20,15 +20,15 @@ public ParcelController(ILogger<ParcelController> logger, IParcelService service
         [HttpGet("parcel/all")]
         public async Task<ActionResult<IEnumerable<Parcel>>> Get()
         {
-            _logger.LogInformation("Received GET request at parcel/all"); //testing purposes
+            _logger.LogInformation("Received GET request at parcel/all Executed {0}->{1}", this.GetType().Name, ControllerContext.ActionDescriptor.ActionName); //testing purposes
```

Comment:
If we have method names in logging, maybe we don't need the "Received GET req...", as the Name of the class is ParcelCon... and method name is Get?

Reply:
Ye, you are right